RDMA IB: Intel fix

adriansuhov opened the pull request:
This commit is consolidating INTEL MPI run by making sure the right variables and binaries are used when running the Intel TCs.

iamshital commented:
Execution is verified
--------------------------
`08/07/2018 13:52:50 : [INFO ] ~~~~~~~~~~~~~~~TEST END : PERF-NETWORK-UDP-THROUGHPUT-MULTICONNECTION-SRIOV~~~~~~~~~~`
`08/07/2018 13:52:50 : [INFO ] CURRENT - PASS    - 1`
`08/07/2018 13:52:50 : [INFO ] CURRENT - FAIL    - 0`
`08/07/2018 13:52:50 : [INFO ] CURRENT - ABORTED - 0`
`08/07/2018 13:52:50 : [INFO ] Checking background cleanup jobs..`
`08/07/2018 13:52:50 : [INFO ] Removed 0 context files.`
`08/07/2018 13:52:50 : [INFO ] Cycle Finished.. TC-59990`
`08/07/2018 13:52:50 : [INFO ] `
`Test Results Summary`
`ICA test run on 08/07/2018 11:43:35`
`ARM Image under test Canonical : UbuntuServer : 18.04-LTS : latest`
`Total Executed TestCases        1 (       1 Pass,        0 Fail,        0 Abort)`
`Total Execution Time(dd:hh:mm) 0:2:9`
`XML file: TestConfiguration`

`PERF-NETWORK-UDP-THROUGHPUT-MULTICONNECTION-SRIOV Execution Time: 129.2 minutes`
`	PERF-NETWORK-UDP-THROUGHPUT-MULTICONNECTION-SRIOV : PASS  `


`Logs can be found at .\TestResults\2018-07-08-11-43-28-2144`


`08/07/2018 13:52:50 : [INFO ] Creating 'Azure-59990-buildlogs.zip' from '.\TestResults\2018-07-08-11-43-28-2144'`
`08/07/2018 13:52:51 : [INFO ] C:\LISAv2\MYZA59990\Azure-59990-buildlogs.zip created successfully.`
`08/07/2018 13:52:51 : [INFO ] Copied : .\report\report_TC-59990.xml --> .\report\report_TC-59990-junit.xml`
`08/07/2018 13:52:51 : [INFO ] Analysing results..`
`08/07/2018 13:52:51 : [INFO ] PASS  : 1`
`08/07/2018 13:52:51 : [INFO ] FAIL  : 0`
`08/07/2018 13:52:51 : [INFO ] ABORT : 0`
`Copying all files back to original working directory: C:\Jenkins-Slaves\azure-slave-2-executor-7\workspace\job-azuretest\job-cloudtest-auto@2.`
`Cleaning up C:\LISAv2\MYZA59990`
`Setting workspace back to original location: C:\Jenkins-Slaves\azure-slave-2-executor-7\workspace\job-azuretest\job-cloudtest-auto@2`
`08/07/2018 13:52:53 : [INFO ] LISAv2 exits with code: 0`

adriansuhov committed:
RDMA IB: Intel fix

This commit is consolidating INTEL MPI run by making sure the right
variables and binaries are used when running the Intel TCs.
